SocialFilter/sections/specific.txt

```diff
@@ -61,6 +61,8 @@ ukranews.com,everydayhealth.com,grid.id,motorplus-online.com,gridoto.com,bolaspo
 !
+keyakizaka46matomemory.net##footer.article-footer ~ div[style]
```

Comment:
It's not necessary to use tag before `>`. The rule below is ok.
```
keyakizaka46matomemory.net##.article-footer ~ div[style]
```

Reply:
Okay.